test/unit/config/karma.unit.conf.cjs

```diff
@@ -73,6 +73,7 @@ module.exports = function (config) {
             resolve: {
                 fallback: {
                     stream: require.resolve('stream-browserify'),
+                    timers: require.resolve('timers-browserify'),
```

Comment:
This dependency is only needed for Karma, and isn't bundled in the final distribution.